src/components/AddForm/AddForm.jsx

```diff
@@ -1,68 +1,86 @@
 import React from 'react';
-import './AddForm.css'
-import CancelButton from './CancelButton/CancelButton'
-import AcceptButton from './AcceptButton/AcceptButton'
-import InputField from './InputField/InputField'
-import { useState } from 'react';
+
+import './AddForm.css';
+import {
+    useState
+} from 'react';
 import PropTypes from 'prop-types';
 
+import Button from '../Button/Button';
+
+import InputField from './InputField/InputField';
+
 function AddForm({
     addTodo,
-    toggleActivity,
-    activeClass
+    toggleVisability,
+    visible
 }) {
 
-    const [inputValue, setInputValue] = useState('')
+    const [inputValue, setInputValue] = useState('');
     const [deadline, setDeadline] = useState('');
 
     const todo = {
         isImportant: true,
         text: 'tasksText',
-        deadline: null,
         isFinished: false,
-    }
+    };
 
     const setTodoDeadline = () => {
         deadline ?
-            todo.deadline = deadline :
-            todo.deadline = new Date(Date.now())
-    }
+            todo.deadline = deadline.toISOString() :
+            todo.deadline = new Date().toISOString();
+    };
 
     const onChangeDeadline = (date) => {
         setDeadline(date);
-    }
+    };
 
     const onClearDeadlineInput = () => {
-        setDeadline('')
-    }
+        setDeadline('');
+    };
 
     const changeTodoText = () => {
-        todo.text = inputValue
-    }
+        todo.text = inputValue;
+    };
 
     const addTodoHandler = () => {
-        addTodo(todo)
-    }
+        addTodo(todo);
+
+    };
 
 
     const onClearInput = () => {
-        setInputValue('')
-    }
+        setInputValue('');
+    };
 
     const onChangeInput = (text) => {
         setInputValue(text);
-    }
-
-    const onAddTodoName = () => {
-        changeTodoText()
-        onClearInput()
-        addTodoHandler()
-    }
+    };
+
+    const onClickSaveButton = () => {
+        toggleVisability({
+            'addbtn': true,
+            'form': false
+        });
```

Comment:
why do you duplicate logic?
can it be situation when you have both variables as true? or false at the same time?